sdk/ai/ai-agents/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.1.0-beta.2 (2025-07-23)
+
+### Bugs Fixed
+
+- Fixed issue with `messages.create` method type error when upload image file.
```

Comment:
If there is a GitHub issue, please provide a link to it. As in "Fix [GitHub issue](link-here) with ..."